src/_data/learnTags.json

```diff
@@ -0,0 +1,24 @@
+{
+	"ethics":	"Ethics",
```

Comment:
If the mapping is defined in an internal json file, content editors will lose the control of adding/removing/editing this list.

I have the same concern about field options that are hardcoded in config.yml. 

I'm not sure how complete the resource collection is and how much chance Lorna or Rachel will modify these options. We probably should run them through the deploy preview, better with Vera and David too, to find out how much flexibility they prefer.

Reply:
Netlify CMS can be configured to edit the contents of this JSON file and the tag select fields on the resources collection can be set up as [relation](https://www.netlifycms.org/docs/widgets/#relation) widgets so that they reference the JSON file directly.

Reply:
I've made this change in 7d3bb0a. Now all resource metadata (focuses, tags, types, tool accessibility issues and tool purposes) can be edited via the Resource Metadata [file collection](https://www.netlifycms.org/docs/collection-types/#file-collections), and all of the fields on resource referencing these values are configured as [relation widgets](https://www.netlifycms.org/docs/widgets/#relation).

Reply:
It's a really good idea to make resource metadata editable. This solves the flexibility issue.

It would be helpful to sort the value list on the resource editing and metadata editing pages?

<img width="1144" alt="resource" src="https://user-images.githubusercontent.com/572877/114729576-1c031880-9d0e-11eb-9a9b-489762af5d85.png">

<img width="788" alt="metadaat" src="https://user-images.githubusercontent.com/572877/114729559-186f9180-9d0e-11eb-8bc2-d71b80fa08a9.png">

Reply:
The order in the Resource editing view will be determined by the order in the Resource Metadata collection. The Resource Types as seen in your screenshot can be reordered by drag and drop, but there is no built-in method of sorting the collection. I can put it in alphabetical order in this pull request, however future changes will need to be put in order manually.

When adding a focus, for example, you can type to search which should help find things even if they aren't in order.

Reply:
Good to know. Thanks, @greatislander.

Reply:
Sorted in 5a4e550.